services/QuillLMS/config/routes.rb

```diff
@@ -368,6 +368,7 @@
       get 'activities/activities_health' => 'activities#activities_health'
       get 'rule_feedback_histories' => 'rule_feedback_histories#by_conjunction'
       get 'rule_feedback_history/:rule_uid' => 'rule_feedback_histories#rule_detail'
+      get 'prompt_health/:activity_id' => 'prompt_feedback_histories#run'
```

Comment:
One thing I just noticed - this should probably be to `rule_feedback_histories#prompt_health`, right?

Reply:
Which makes me feel like maybe we should also have a controller test.